petastorm/reader.py

```diff
@@ -152,9 +150,9 @@ def make_reader(dataset_url,
         reader_pool = ThreadPool(workers_count, results_queue_size)
     elif reader_pool_type == 'process':
         if pyarrow_serialize:
-            serializer = PyArrowSerializer()
-        else:
-            serializer = PickleSerializer()
+            warnings.warn("pyarrow_serializer was deprecated and will be removed in future versions. "
+                          "The argument no longer has any effect.")
```

Comment:
Isn't it more accurate to say `was deprecated and removed. The argument no longer has any effect.`.

S

Reply:
Strictly speaking we didn't remove it from the API. It does not have any effect anymore and that's mentioned in the message. If we would remove the argument, the existing code would break.